redaxo/src/addons/phpmailer/lib/cronjob.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * PHPMailer Addon.
+ *
+ * @author markus[dot]staab[at]redaxo[dot]de Markus Staab
+ *
+ * @package redaxo\phpmailer
+ */
+
+class rex_cronjob_mailer_purge extends rex_cronjob
+{
+    /**
+     * purgeMailarchive.
+     *
+     * @return int
+     */
+    protected function purgeMailarchive(int $days = 7, string $dir = '', int $log = 0)
```

Comment:
Wozu ist `$log` ein Parameter? Könnte doch einfach eine Variable sein, die in der Methode oben mit `0` initialisiert wird.

Reply:
Ich hatte das versucht. Dann kam aber ein falsches Ergebnis raus. Kannst du mir da noch mal helfen?

Reply:
Nochmal nachgeschaut ... ich hatte wohl ursprünglich was anderes vor und wollte $log an die function übergeben. 
Stimmt es passt so jetzt. Ich habe $log nun am Anfang auf 0 gesetzt. Danke für den Hint.